### ISSUE TITLE
The Sink .import() method doesn't provide feedback

### ISSUE THREAD

**Issue by jacoscaz** [rdfjs/representation-task-force#106]
The `.import(stream.Readable stream)` method from the `Sink` interface does not seem to provide any feedback regarding the state of the import operation as it does not return an `events.EventEmitter` and/or take a callback as an argument.

In `node-quadstore` I've opted to have it return an instance of `events.EventEmitter` to make it consistent with `remove()`. Should I make a PR for this?

**Comment by elf-pavlik** [comment]
Looking at your example https://github.com/beautifulinteractions/node-quadstore#rdfstoreprototypeimport

```js
const readableStream; // A stream.Readable of Quad() instances

store.import(readableStream)
  .on('error', (err) => {})
  .on('end', () => {});
```
I think currently one needs to handle events on that `readableStream` passed to `import`

```js
const readableStream // A stream.Readable of Quad() instances
  .on('error', (err) => {})
  .on('end', () => {})

store.import(readableStream)
```

@RubenVerborgh have already implemented `import` interface in N3.js ?

**Comment by RubenVerborgh** [comment]
Hmm, maybe it's not because the input stream has been read, that all triples have been processed.

Haven't implemented that yet in N3.js, I'm really behind. Hoping for April.

**Comment by jacoscaz** [comment]
I agree with @RubenVerborgh. The `end` event should signal the actual termination of the import procedure, which might happen some time after the source stream has ended. The `error` event, on the other hand, might _also_ relay errors from the source stream, although I'm not 100% sure about that.

**Comment by elf-pavlik** [comment]
> The end event should signal the actual termination of the import procedure, which might happen some time after the source stream has ended.

Makes sense, @jacoscaz would like to make another PR for it?

**Comment by bergos** [comment]
I had the same problem of the missing feedback and implemented it with an EventEmitter: https://github.com/rdf-ext/rdf-sink/blob/develop/index.js#L9

**Comment by elf-pavlik** [comment]
addressed in #110
